pkg/addon/controllers/addonconfiguration/graph.go

```diff
@@ -145,26 +157,40 @@ func (g *configurationGraph) addAddonNode(mca *addonv1alpha1.ManagedClusterAddOn
 func (g *configurationGraph) addPlacementNode(
 	installStrategy addonv1alpha1.PlacementStrategy,
 	installProgression addonv1alpha1.InstallProgression,
-	clusters []string,
-) {
+	placement *clusterv1beta1.Placement,
+	pdTracker *clusterv1beta1.PlacementDecisionClustersTracker,
+) error {
 	placementRef := installProgression.PlacementRef
 	installConfigReference := installProgression.ConfigReferences
 
+	// init pdTracker clusters
```

Comment:
enhance the comment here, it is not quite clear the intention

Reply:
remove initClusters func and add comments // refresh and get existing decision clusters